rn/Teacher/ios/Teacher/TeacherTabBarController.swift

```diff
@@ -34,10 +34,15 @@ class TeacherTabBarController: UITabBarController {
             paths.firstIndex(of: $0)
         } ?? 0
         tabBar.useGlobalNavStyle()
-
+        NotificationCenter.default.addObserver(self, selector: #selector(checkForPolicyChanges), name: UIApplication.didBecomeActiveNotification, object: nil)
```

Comment:
We should remove this observer, otherwise on logout/user change the tab bar controller will leak.